packages/grafana-ui/src/components/Layout/Stack/Stack.tsx

```diff
@@ -2,18 +2,27 @@ import React from 'react';
 
 import { ThemeSpacingTokens } from '@grafana/data';
 
-import { Direction, Flex } from '../Flex/Flex';
-import { ResponsiveProp } from '../utils/responsiveness';
+import { Flex } from '../Flex/Flex';
+
+import { HorizontalStack } from './HorizontalStack';
```

Comment:
You've created a cyclic dependency here, which can cause all sorts of issues. 

Instead create an index.ts that exports both Stack and HorizontalStack